src/main/server.ts

```diff
@@ -68,21 +116,9 @@ class JupyterServer {
             this._info.url = `http://localhost:${appConfig.jlabPort}`;
             this._info.token = appConfig.token;
 
-            // note: traitlets<5.0 require fully specified arguments to
-            // be followed by equals sign without a space; this can be
-            // removed once jupyter_server requires traitlets>5.0
-            this._nbServer = execFile(this._info.environment.path, [
-                '-m', 'jupyterlab',
```

Comment:
`-m` is not a great daily driver, as it effectively adds cwd to `PYTHONPATH`... it's very easy for local paths to clobber Important Python Stuff.

Reply:
@bollwyvl calling `jupyter lab` is failing on Windows with error `Error executing Jupyter command 'lab': [WinError 5] Access is denied `. that's why we have been using `python -m`